core/lib/generators/solidus/install/app_templates/authentication/none.rb

```diff
@@ -0,0 +1,23 @@
+initializer 'solidus_authentication.rb', <<~RUBY
+  module SolidusAuthenticationControllerHelper
+    def spree_current_user
+      Spree::LegacyUser.first ||
+        Spree::LegacyUser.create!(spree_roles: [
```

Comment:
Why do we want to create a user if there's none (at least without checking flags like `--sample`)?

Reply:
The intent is to ensure the resulting store is functional and can be explored, with this setup I was able to complete a checkout. 

But if you think it's too much I'm ready to remove it as I was also not sure it was worth it.

Reply:
I understand that `none` means: "We are not going to install an authentication system for you now, but you'll need to do it yourself". If I understand correctly, having a functional Solidus without authentication is impossible. If I'm correct, we could:
- Warn when `none` is selected, or be more explicit in the prompt message.
- Raise a meaningful error here in `spree_curent_user`.

Reply:
Ok, I pushed a new version that will both allow complete access to /admin and the ability to place orders with some inline documentation in the initializer that should give the ropes of what to do to roll your own authentication.